airflow/jobs/scheduler_job.py

```diff
@@ -1408,6 +1443,40 @@ def _send_sla_callbacks_to_processor(self, dag: DAG) -> None:
         )
         self.executor.send_callback(request)
 
+    @provide_session
+    def _fail_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
+        """
+        Mark tasks stuck in queued for longer than `task_queued_timeout` as failed.
```

Comment:
In the doc string here, we are using `task_queued_timeout` but self._task_queued_timeout is a max of many other configs